.azure/pipelines/report-green.yml

```diff
@@ -0,0 +1,37 @@
+# This CI job only runs on PRs where all other jobs are skipped.
+# This allows Build Analysis to report green. Without this, no jobs would run,
+# causing Build Analysis to hang indefinitely (or until someone commented "ba-g {justification}" on the PR).
+
+# Only run this on PRs
+trigger: none
+# Run for all branches, only on paths that no-op other jobs
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - '*'
+  paths:
+    include:
+    - .devcontainer/*
+    - .github/*
+    - .vscode/*
+    - docs/*
+    - '**/*.md'
```

Comment:
markdown lint doesn't cover these?

Reply:
It does, but that's a github action - Build Analysis only looks at AzDO jobs